<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 40, 51);</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>493</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>100</number>
    </property>
    <property name="rightMargin">
     <number>100</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(69, 162, 158);
font: 75 12pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>Keyboard trainer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 10pt &quot;Trebuchet MS&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 8pt &quot;Trebuchet MS&quot;;
</string>
      </property>
      <property name="text">
       <string>Введите ник (используйте латиницу)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 10pt &quot;Trebuchet MS&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 8pt &quot;Trebuchet MS&quot;;
</string>
      </property>
      <property name="text">
       <string>Введите пароль (используйте любые символы)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 0, 0);
font: 75 8pt &quot;Trebuchet MS&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 10pt &quot;Trebuchet MS&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
font: 75 8pt &quot;Trebuchet MS&quot;;
</string>
      </property>
      <property name="text">
       <string>Введите почту</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>sign up</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
      </property>
      <property name="text">
       <string>back to main window</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
